cloudhub-2/modules/ROOT/pages/ps-gather-setup-info.adoc

```diff
@@ -385,6 +405,17 @@ You cannot use the following reserved CIDR blocks:
 169.254.169.252/30
 ----
 
+Anypoint Platform has a list of CIDRs as an internal CIDR pool for provisioning private space networks. The MuleSoft CIDR pool is subject to change. You can configure the following as reserved CIDR blocks, but you cannot use them as private space network CIDRs:
+
+----
+100.64.0.0/16
+100.66.0.0/16
+100.67.0.0/16
+100.68.0.0/16
+----
```

Comment:
@tonylsw the four entries are down here.

Reply:
yes, they look good here.